lib/routes/misskey/user-timeline.ts

```diff
@@ -0,0 +1,41 @@
+import { Route } from '@/types';
+import utils from './utils';
+import { config } from '@/config';
+import ConfigNotFoundError from '@/errors/types/config-not-found';
+
+export const route: Route = {
+    path: '/users/notes/:username',
+    categories: ['social-media'],
+    example: '/misskey/users/notes/@support@misskey.io',
+    parameters: { username: 'misskey username format, like @support@misskey.io' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'User timeline',
+    maintainers: ['siygle'],
+    handler,
+};
+
+async function handler(ctx) {
+    const username = ctx.req.param('username');
+    const [, pureusername, site] = username.match(/@(\w+)@(\w+\.\w+)/) || [];
+    if (!pureusername || !site) {
+        throw new ConfigNotFoundError('Provide a valid Misskey username');
+    }
+    if (!config.feature.allow_user_supply_unsafe_domain && !utils.allowSiteList.includes(site)) {
+        throw new ConfigNotFoundError(`This RSS is disabled unless 'ALLOW_USER_SUPPLY_UNSAFE_DOMAIN' is set to 'true'.`);
+    }
+
+    const { account_data } = await utils.getUserTimelineByUsername(pureusername, site);
```

Comment:
`camelCase`

Reply:
Sorry, just double confirm about this issue. Is RSSHub asking the camcelCase rule by default? I folk the codebase from other services under routes directly, and seems it did not follow `camelCase` as well.

Reply:
https://docs.rsshub.app/joinus/advanced/script-standard
Rename the destructed variable